lib/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @fileoverview Shareable eslint plugin for use by eslint-config-silvermine
+ */
+
+'use strict';
+
+var fluentChaining = require('./rules/fluent-chaining'),
+    arrayIndentation = require('./rules/array-indentation'),
+    callIndentation = require('./rules/call-indentation'),
+    noMultipleFunctionDeclarations = require('./rules/no-multiple-func-decl');
+
+module.exports.rules = {
+   'fluent-chaining': fluentChaining,
+   'array-indentation': arrayIndentation,
+   'call-indentation': callIndentation,
+   'no-multiple-func-decl': noMultipleFunctionDeclarations
```

Comment:
Let's rename `no-multiple-func-decl` to something more clear. As it's named now, it sounds like you can't declare multiple functions. I would like it to be named `no-multiple-inline-functions`. This means the renaming of the files in this PR as well - just commenting on it in this one location, though.